docker-compose.yml

```diff
@@ -130,12 +143,15 @@ services:
     depends_on:
       - postgres
   guppy-service:
-    image: "quay.io/cdis/guppy:2020.09"
+    image: "quay.io/cdis/guppy:2021.03"
     container_name: guppy-service
     networks:
       - devnet
     volumes:
       - ./Secrets/guppy_config.json:/guppy/guppy_config.json
+      - ./scripts/wait_for_esproxy.sh:/usr/bin/wait_for_esproxy.sh:ro
+    entrypoint: /usr/bin/wait_for_esproxy.sh
```

Comment:
Since the default build doesn't handle ETL from scratch guppy still crashes, but this at least gets it a little closer to stable. It's at least waiting for elastic search now. It changes this error...

```
guppy-service        | [14:05:45] INFO: [ES.initialize] getting mapping from elasticsearch...
guppy-service        | (node:7) DeprecationWarning: You are using a version of Node.js that will reach EOL in April 2021. The support for this version will be dropped in 7.13. Please refer to https://ela.st/nodejs-support for additional information.
guppy-service        | [14:05:45] ERROR: [ES] elasticsearch cluster at http://esproxy-service:9200 is down!
guppy-service        | (node:7) UnhandledPromiseRejectionWarning: Error: [ES.initialize] error getting mapping from ES index "etl": connect ECONNREFUSED 192.168.80.2:9200
guppy-service        |     at client.indices.getMapping.then.err (/guppy/dist/server/es/index.js:171:13)
guppy-service        |     at process._tickCallback (internal/process/next_tick.js:68:7)
guppy-service        | (node:7) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
guppy-service        | (node:7) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
guppy-service exited with code 0
```

...to this error...

```
guppy-service        | [14:00:54] INFO: [ES.initialize] getting mapping from elasticsearch...
guppy-service        | (node:1) DeprecationWarning: You are using a version of Node.js that will reach EOL in April 2021. The support for this version will be dropped in 7.13. Please refer to https://ela.st/nodejs-support for additional information.
guppy-service        | (node:1) UnhandledPromiseRejectionWarning: Error: [ES.initialize] error getting mapping from ES index "etl": index_not_found_exception
guppy-service        |     at client.indices.getMapping.then.err (/guppy/dist/server/es/index.js:171:13)
guppy-service        |     at process._tickCallback (internal/process/next_tick.js:68:7)
guppy-service        | (node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
guppy-service        | (node:1) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
guppy-service        | [14:00:54] INFO: [ES] connected to elasticsearch at http://esproxy-service:9200.
guppy-service exited with code 0

Reply:
Having `index_not_found_exception` in the logs is definitely an improvement 😄